bioblend/galaxy/invocations/__init__.py

```diff
@@ -181,6 +216,42 @@ def get_invocation_report(self, invocation_id):
         url = self._make_url(invocation_id) + '/report'
         return self._get(url=url)
 
+    def get_invocation_report_pdf(self, invocation_id, file_path, chunk_size=CHUNK_SIZE):
+        """
+        Get a PDF report for an invocation.
+
+        :type invocation_id: str
+        :param invocation_id: Encoded workflow invocation ID
+
+        :type file_path: str
+        :param file_path: Path to save the file under
+
+        :rtype: bytes
+        :return: The invocation report.
+        """
+        url = self._make_url(invocation_id) + '/report.pdf'
+        try:
+            r = self.gi.make_get_request(url, stream=True, json=False)
```

Comment:
I don't think `json=Falsee` is needed?

```suggestion
            r = self.gi.make_get_request(url, stream=True)
```

Reply:
Okay